### PR TITLE
Fix overflow memory tests for recorders

### PR DESCRIPTION
**Pull request by h-mayorquin**
The other margin was too small as indicated by an error during https://github.com/catalystneuro/neuroconv/pull/33

This PR should fix it.

### CODE REVIEW

File: tests/test_internals/test_si.py [hunk 1]
```diff
@@ -700,12 +700,14 @@ def test_non_iterative_write(self):
...
         element_size_in_bytes = dtype.itemsize
         num_channels = self.test_recording_extractor.get_num_channels()
+        num_frames = self.test_recording_extractor.get_num_frames()
 
         available_memory_in_bytes = psutil.virtual_memory().available
         frame_size_in_bytes = element_size_in_bytes * num_channels
-        available_memory_in_bytes_plus_one_frame = available_memory_in_bytes + frame_size_in_bytes
 
-        num_frames_to_overflow = available_memory_in_bytes_plus_one_frame / (element_size_in_bytes * num_channels)
+        excess_in_bytes = frame_size_in_bytes * int(num_frames / 2)  # 50 % excess frames
```
Comment (CodyCBakerPhD) [inline tests/test_internals/test_si.py:708]: Doesn't this result in fewer total frames in the mock?

Comment (h-mayorquin) [reply tests/test_internals/test_si.py:708]: excess_in_bytes is what is on top of the estimated available memory, there is a sum down there.

Comment (CodyCBakerPhD) [reply tests/test_internals/test_si.py:708]: Oh, I see - you didn't originally have the `excess_in_bytes` term in the overflow equation.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin https://github.com/catalystneuro/neuroconv/runs/7501021543?check_suite_focus=true

Apparently, it's not enough XD maybe try an excess of 3x the original size.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by h-mayorquin** [comment]
> @h-mayorquin https://github.com/catalystneuro/neuroconv/runs/7501021543?check_suite_focus=true
> 
> Apparently, it's not enough XD maybe try an excess of 3x the original size.

I was doing it wrong. I was adding 1.5, 2.5, and 3.0 extra frames (!). I just added now 1.5 of what is available in memory I think that should work.

**Pull request merged**